cogs/misc.py

```diff
@@ -420,6 +429,15 @@ async def parse(self, ctx, *, time_and_arg: converters.HumanTime(other=True)):
                            f'Other argument: {time_and_arg.other}\n'
                            f'Delta: {(time - pendulum.instance(ctx.message.created_at)).in_words()}')
 
+    @commands.command()
+    async def someone(self, ctx, *, text)
+        """
+        Repeats the message but with a random member's name after :someone
```

Comment:
This command will not be invoked as `:someone` but rather `{prefix}someone`, where prefix is whatever the bot's prefix in that guild with default of `>`. Either use the default prefix of `>` or just mention `{prefix}`